youtubedl/gui.py

```diff
@@ -64,8 +69,55 @@ def enterURL(self):
             230, 230, Qt.KeepAspectRatio, Qt.FastTransformation)
         self.labelThumbnail.setPixmap(pixmap)
 
+        # Populate combo box
+        self.populateComboBox()
+
+        # enable download button
+        self.btnDownload.setEnabled(True)
+
+        # debug information
+        self.logger.info(f"URL: {self.ytube.url}")
+        self.logger.info(f"Video Title: {self.ytube.videoTitle}")
+        self.logger.info(
+            f"Video Thumbnail: {self.ytube.videoThumbnail}")
```

Comment:
Don't need these lines. The YouTubeVideo class already logs this information.

Reply:
Ok.I did these before you have done all the logging changes.. Will remove it later

Reply:
fixed